### ISSUE TITLE
Machines stops after leave chunk and back.

### ISSUE THREAD

**Issue by BaalEvan** [McJty/DeepResonance#119]
Every time you leave Base where the machines are placed with Deep Resonance, after returning cease to operate.

As the screen below the lava is below the smelter and is hooked to power but still does not start to work. I have to destroy it and put again to start the work. The same with Valve, Purifier and Crystalizer (the biggest problem is with the Crystalizer because after the destruction he loses his progress)

Minecraft Version: 1.10.2
Deep Resonance Version: 1.10.2-1.2.7
ElecCore Version: 1.9.4-1.5.311
Forge version: 1.10.2-12.18.1.2094

Map is on Standalone server

I think it's starts after update from 1.2.5 and forge 2076 ( i can upload modpack with map ) :)

Edit: No problem on SSP

Edit2: After server restart machines works again.

![javaw_2016-10-03_03-11-48](https://cloud.githubusercontent.com/assets/1843494/19025304/64e21868-8917-11e6-8491-8aa39518fcb0.png)

![2016-10-03_03-17-55](https://cloud.githubusercontent.com/assets/1843494/19025355/05f9f04a-8918-11e6-875a-1f605c9f9d66.gif)

**Comment by McJty** [comment]
What...

**Comment by BaalEvan** [comment]
Look at the video:

https://drive.google.com/open?id=0Byz1G0PvYk3Fa093cC1nWTBmMXc

Crystalizer works -> Leave base and fly away -> Go back -> Crystalizer's progress freeze

**Comment by McJty** [comment]
but that's in specactor mode. As far as I now spectactor mode does not load chunks. i.e. tile entities will not activate if you are in spectator mode

**Comment by BaalEvan** [comment]
I'm in Creative mode not Spectator ;)

**Comment by McJty** [comment]
You were flying through things. That can't be done in creative mode. That was definitely spectator mode

**Comment by BaalEvan** [comment]
What? In 1:00 you can see landing on the roof. And at the beginning 0:04 i hit door frame.

**Comment by McJty** [comment]
Perhaps you sent me the wrong video because in the video I got I see someone flying through blocks all the time. Exploring caves and such.

**Comment by McJty** [comment]
(sorry, accidently closed)

**Comment by McJty** [comment]
Also I can see nothing related to Deep Resonance on that video

**Comment by BaalEvan** [comment]
Yeah on the beginning i sent wrong but i edit this comment can you refresh browser tab?

**Comment by BaalEvan** [comment]
Also you should have two video links.

**Comment by McJty** [comment]
Ok, that's strange. What other mods do you have installed? I have never seen or have this reported before and I cannot reproduce this locally

**Comment by BaalEvan** [comment]
If you want test on my Server here is address: 

s23.hostmc.pl:15209

[modlist.txt](https://github.com/McJty/DeepResonance/files/506113/modlist.txt)

Exported Curse Profile: (configs + mods)
https://drive.google.com/open?id=0Byz1G0PvYk3FOXVYdnE3dXVOaXc

**Comment by McJty** [comment]
Testing outside a dev env doesn't really help. I can't do any really debugging except notice the problem and that I can already see in your video. BTW what about other machines? Did you test with machines from other mods to see what they do? Preferably machines placed in the same chunk as the DR machines

**Comment by BaalEvan** [comment]
So maybe i send you my world? 
I have liquid monitor from RFTools, Quantum Entangloporter, Lava Tank, Cables and InductionMatrix from Mekanism, in same building. Everything works. Laser from DR works too.

Generator from DR is in basement but i'm not sure it's work or not.

Chunk Screen
![javaw_2016-10-03_13-52-58](https://cloud.githubusercontent.com/assets/1843494/19036544/be5dde28-8970-11e6-98ac-cc5e0ece1349.jpg)

**Comment by BaalEvan** [comment]
I place Alloy Smelter from EnderIO in same chunk and it's work too

**Comment by McJty** [comment]
yes, if you can upload modpack+map that might help

**Comment by BaalEvan** [comment]
Link Here:
https://drive.google.com/open?id=0Byz1G0PvYk3FSjBKeEdmX2Y5dHM

**Comment by BaalEvan** [comment]
I recreate bug on new world (same SMP server) funny thing: even lava image is frozen (gif below)

![2016-10-03_18-03-26](https://cloud.githubusercontent.com/assets/1843494/19044262/c24cb810-8993-11e6-946a-a94d9609e1fe.gif)

**Comment by SharkWipf** [comment]
I'm having the same issue on my custom 1.10.2 modpack server (https://minecraft.curseforge.com/projects/blockheadism).
Whenever chunks are unloaded, DeepResonance machines (and no other machines, mind you) stop working as shown in this thread.
If you haven't found the issue yet, I assume it's not fixed yet in the latest version, but sadly I can't really test that yet because of issue #127.
If you want to I can set up a clone of my server and give you OP and SSH access on it, not really a development environment but at least you could test it live then and upload new versions as needed.

This issue existed at least since I first set up this server on September 23 and has persisted in every version since.

**Comment by SharkWipf** [comment]
This bug might have been fixed in the latest version, haven't been able to test extensively yet but machines that were stopped before that I was too lazy to replace suddenly started working again. 
Will report back if it stops working again.

**Comment by SharkWipf** [comment]
Can confirm this bug still happens on the latest version of everything, that is, DeepResonance, Forge, CompatLayer and anything else installed on our modpack. Putting down a chunkloader "fixes" it until the server restarts.

**Comment by McJty** [comment]
Which versions exactly? Also what version of MC?

**Comment by McJty** [comment]
Also what other mods are there?

**Comment by SharkWipf** [comment]
Whoops, should have mentioned yeah. 
- Minecraft 1.10.2
- Forge version 1.10.2-12.18.2.2179
- Deep Resonance version 1.11-1.3.1beta1 (marked as 1.10.2 compatible on Curse so I assume that's the right one, this bug predates 1.11 anyway)
- CompatLayer 1.10-0.0.39beta
- 1 non-Curse mod, Unborked, but the issue happened before inclusion of this mod.
- All other mods are listed [here](https://web.blockheadism.net/modlist.html) (w/o versions), [here](https://web.blockheadism.net/modlist-ls.txt) (with versions, ls output), or [here](https://minecraft.curseforge.com/projects/blockheadism/relations/dependencies) (Curse modpack link).

There's a couple of client-side only mods on the server, I'm aware, I don't think those are the cause. It's easier to upload all client-mods to the server than to filter 'em out every update, but if you want I can remove 'em and see if the issue persists. 
My offer to give you SSH access to a clone of the server still stands btw.

**Comment by aaronmallen** [comment]
This is also happening with the valves. Blew through 4.5K diamonds because a valve got stuck in a similar fashion.

**Comment by Schmocky** [comment]
Currently playing the AoE Modpack (Version 1.0.5) on Minecraft Version 1.10.2 and it seems like I am having the same issue. Machines just stop working when I leave the server. And after joining I have to replace all of them to get them to work.

**Comment by McJty** [comment]
To @Schmocky , What version of DR and ElecCore?

**Comment by Schmocky** [comment]
@McJty DR: 1.4.5 ElecCore: 1.7.431
